modules/hardware/definitions/dell-latitude/dell-latitude-7230.nix

```diff
@@ -0,0 +1,155 @@
+      ];
+      evdev = [
+        # /dev/input/by-path/platform-INTC1070:00-event /dev/input/by-path/platform-PNP0C14:02-event
+      ];
+    };
+  };
+
+  # Main disk device
+  disks = {
+    disk1.device = "/dev/nvme0n1";
+  };
+
+  # Network devices for passthrough to netvm
+  network = {
+    pciDevices = [
+      {
+        # Network controller: Intel Corporation Alder Lake-P PCH CNVi WiFi (rev 01)
+        name = "wlp0s5f0";
+        path = "0000:00:14.3";
+        vendorId = "8086";
+        productId = "51f0";
+        # Detected kernel driver: iwlwifi
+        # Detected kernel modules: iwlwifi
+      }
+    ];
+    kernelConfig = {
+      stage1.kernelModules = [];
+      stage2.kernelModules = ["iwlwifi"];
+      kernelParams = [];
+    };
+  };
+
+  # GPU devices for passthrough to guivm
+  gpu = {
+    pciDevices = [
+      {
+        # VGA compatible controller: Intel Corporation Alder Lake-UP4 GT2 [Iris Xe Graphics] (rev 0c)
+        name = "gpu0";
+        path = "0000:00:02.0";
+        vendorId = "8086";
+        productId = "46aa";
+        # Detected kernel driver: i915
+        # Detected kernel modules: i915
+      }
+    ];
+    kernelConfig = {
+      stage1.kernelModules = ["i915"];
+      stage2.kernelModules = [];
+      kernelParams = ["earlykms"];
+    };
+  };
+
+  # Audio device for passthrough to audiovm
+  audio = {
+    pciDevices = [
+      {
+        # ISA bridge: Intel Corporation Alder Lake LPC Controller (rev 01)
+        name = "snd0-0";
+        path = "0000:00:1f.0";
+        vendorId = "8086";
+        productId = "5187";
+        # Detected kernel driver:
+        # Detected kernel modules:
+      }
+      {
+        # Serial bus controller: Intel Corporation Alder Lake-P PCH SPI Controller (rev 01)
+        name = "snd0-1";
+        path = "0000:00:1f.5";
+        vendorId = "8086";
+        productId = "51a4";
+        # Detected kernel driver: intel-spi
+        # Detected kernel modules: spi_intel_pci
+      }
+      {
+        # Audio device: Intel Corporation Alder Lake Smart Sound Technology Audio Controller (rev 01)
+        name = "snd0-2";
+        path = "0000:00:1f.3";
+        vendorId = "8086";
+        productId = "51cc";
+        # Detected kernel driver: snd_hda_intel
+        # Detected kernel modules: snd_hda_intel,snd_sof_pci_intel_tgl
+      }
+      {
+        # SMBus: Intel Corporation Alder Lake PCH-P SMBus Host Controller (rev 01)
+        name = "snd0-3";
+        path = "0000:00:1f.4";
+        vendorId = "8086";
+        productId = "51a3";
+        # Detected kernel driver: i801_smbus
+        # Detected kernel modules: i2c_i801
+      }
+    ];
+    kernelConfig = {
+      stage1.kernelModules = [];
+      stage2.kernelModules = ["i2c_i801" "snd_hda_intel" "snd_sof_pci_intel_tgl" "spi_intel_pci"];
```

Comment:
`lsmod` shows that these drivers have many dependencies so this list is incomplete and likely will not work without all the dependent drivers. either document that this is just to show the primary drivers, and whoever is making the minimal module list will have to include the others or you would have to traverse the `lsmod` or `modinfo` to extract them all

Reply:
There are a number of warnings already (e.g., in --help), where do you want to have this? As comment above?